apps/modernization-ui/src/design-system/inPageNavigation/InPageNavigation.module.scss

```diff
@@ -0,0 +1,42 @@
+@use 'styles/colors';
+@use 'styles/borders';
+
+nav {
+    position: sticky;
+    top: 0;
+
+    .navTitle {
+        padding: 0;
+        font-size: 0.875rem;
+        font-weight: 700;
+    }
+
+    .navOptions {
+        display: flex;
+        flex-direction: column;
+        margin-top: 1rem;
+        @include borders.bordered('border-left');
+
+        a {
+            padding: 0.5rem 1rem;
+            text-decoration: none;
+            border-left: 4px solid transparent !important;
+
+            &:focus {
+                outline: none !important;
+                background-color: transparent !important;
+            }
+
+            &.visible {
```

Comment:
question (non-blocking): Aren't all of the sections visible? Is there a more descriptive name?